Xamarin.FacebookSlim.Sample/ViewController.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using Foundation;
+using SkiaSharp;
+using SkiaSharp.Views.iOS;
+using UIKit;
+using Xamarin.FacebookSlim;
+
+namespace Sample
+{
+    public partial class ViewController : UIViewController
+    {
+        private SKBitmap _bitmap;
+
+        public ViewController(IntPtr handle) : base(handle)
+        {
+        }
+
+        public override void ViewDidLoad()
+        {
+            base.ViewDidLoad();
+            // Perform any additional setup after loading the view, typically from a nib.
+
+            var image = new UIImageView
+            {
+                TranslatesAutoresizingMaskIntoConstraints = false
+            };
+
+            Add(image);
+
+            UIButton button = AddButton("Placeholder");
+
+            button.TouchUpInside += (s, e) =>
+            {
+                var logonManager = LoginManagerSlim.Shared;
+                logonManager.LoginWithViewController(this, delegateHandler);
+            };
+
+            var skiaView = new SKCanvasView
+            {
+                TranslatesAutoresizingMaskIntoConstraints = false
+            };
+            skiaView.PaintSurface += OnPaintSurface;
+            Add(skiaView);
+
+            UIButton button2 = AddButton("Streamed image");
```

Comment:
is this an artifact form the copied nuke bindings sample?

Reply:
removed